test/cache/json_writer_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class Measured::Cache::JsonWriterTest < ActiveSupport::TestCase
+  class JsonTestWithWriter < Measured::Cache::Json
+    prepend Measured::Cache::JsonWriter
```

Comment:
I tried using `ActiveSupport::Testing::Isolation` here but it broke a few other things. Settled for subclassing and prepending instead. It's trying to test the specialization of the superclass, not the implementing class though.